index.js

```diff
@@ -1,47 +1,66 @@
 var transit = require('transit-js');
 var Immutable = require('immutable');
 
-var reader = transit.reader('json', {
-  mapBuilder: {
-    init: function() {
-      return {};
-    },
-    add: function(m, k, v) {
-      m[k] = v;
-      return m;
-    },
-    finalize: function(m) {
-      return m;
-    }
-  },
-  handlers: {
-    iM: function(v) {
-      var m = Immutable.Map().asMutable();
-      for (var i = 0; i < v.length; i += 2) {
-        m = m.set(v[i], v[i + 1]);
+function recordName(record) {
```

Comment:
This doesn't quite match https://github.com/facebook/immutable-js/blob/7c4483e30e8b9528562f6ef34f66bee6bfb3538d/src/Record.js#L150 - perhaps we should request that this become a public method?